changelog/unreleased/issue-2345

```diff
@@ -1,8 +1,8 @@
 Bugfix: Make cache crash-resistant and usable by multiple concurrent processes
 
-The restic cache directory ($RESTIC_CACHE_DIR) could end up in a broken state
+The restic cache directory (`RESTIC_CACHE_DIR`) could end up in a broken state
 in the event of restic (or the OS) crashing. This is now less likely to occur
-as files are downloaded to a temporary location before being moved in place.
+as files are downloaded to a temporary location before being moved into place.
```

Comment:
When I wrote this, I was thinking of the expression "put in place", but I'm not sure if "move in place" can be formed by analogy. Maybe

```suggestion
as files are downloaded to a temporary location before being moved to their
proper location.
```